crates/ty_python_semantic/resources/mdtest/assignment/annotations.md

```diff
@@ -366,7 +285,7 @@ class Z[T]:
 
 z1: Z[Any] = Z(1)
 # TODO: This should reveal `Z[Any]`.
```

Comment:
Do we have a task open for this TODO? It does seem like inference of generic parameters is a case where we should fully respect the type context.

Reply:
This is fixed by https://github.com/astral-sh/ruff/pull/21442.